settings/teams.json

```diff
@@ -46,5 +46,29 @@
                 "AI_file": "AI_file4.py"
             }
         ]
+    },
+    "TeamZach": {
```

Comment:
Shouldn't these teams get a directory in the teams folder?

Reply:
You're so right, fixing now.